### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by eanplatter**
Our README had some out of date instructions.

### CODE REVIEW

**Comment by athurman** [comment]
:heart_eyes_cat: Looks good!

File: README.md [hunk 1]
```diff
@@ -19,39 +19,40 @@ var TodoList = tapFlux.createModule();
 
 TodoList
 
-  .setConstants({
-    ADD_ITEM: 'ADD_ITEM',
-    REMOVE_ITEM: 'REMOVE_ITEM'
+  .createActionTypes({
+    ADD_ITEM: '',
+    REMOVE_ITEM: ''
```
Comment (levithomason) [inline README.md]: How about an inline comment here about the keys being mirrored recursively?  Since, the convention is not very well known.

Comment (levithomason) [inline README.md]: Scratch that, it should go in the How it works below.

**Comment by levithomason** [comment]
How it works section has:

```js
TodoList.setConstants({
  ADD_ITEM: 'ADD_ITEM',
  REMOVE_ITEM: 'REMOVE_ITEM'
});
```

Aren't those mirrored for the user now?

File: README.md [hunk 2]
```diff
@@ -73,13 +74,13 @@ TodoList.setConstants({
 
 Nothing is exported because all parts of a module are accessed through the module:
 
-#### setConstants
+#### createActionTypes
 
-Attaches constants to `TodoList.constants`.
+Attaches constants to `TodoList.actionTypes`.
```
Comment (levithomason) [inline README.md]: Typo.  Attached "actionTypes", not "constants" :)

File: README.md [hunk 3]
```diff
@@ -19,39 +19,40 @@ var TodoList = tapFlux.createModule();
...
         data: index
-      })
+      });
     }
   })
 
   .createStore('itemStore', {
-    initialize: function() {
-      this.items = [];
+    items = [],
```
Comment (levithomason) [inline README.md]: I think you meant `items: []` here (no equal sign).

**Comment by levithomason** [comment]
Road map section is all done :D  woop woop!  can be updated or flat deleted!

**Comment by levithomason** [comment]
In How it works > createStore, it might be really helpful to have a note about the magical action name matching to store setters.

**Pull request merged**